mitremap-notebook/README.md

```diff
@@ -0,0 +1,202 @@
+# MitreMap - Inferring MITRE Technique from Threat Intel Data
+
+## Table of Contents
+1. [Motivation](#motivation)
+2. [MITRE ATT&CK Framework](#mitre-attck-framework)
+3. [Goals of the MitreMap Notebook](#goals-of-the-mitremap-notebook)
+4. [One-Time Setup](#one-time-setup)
+    - [1. Creating a virtual environment](#1-creating-a-virtual-environment)
+    - [2. Downloading model artifacts](#2-downloading-model-artifacts)
+5. [Input Parameters](#input-parameters)
+6. [Demo](#demo)
+    - [1. MITRE Technique Inference for Threat Intel Data, WITH Model Explainability](#1-mitre-technique-inference-for-threat-intel-data-with-model-explainability)
+    - [2. MITRE Technique Inference for Threat Intel Data, WITHOUT Model Explainability](#2-mitre-technique-inference-for-threat-intel-data-without-model-explainability)
+7. [Use the MitreMap Notebook outside of Sentinel Notebooks](#7-use-the-mitremap-notebook-outside-of-sentinel-notebooks)
+<br><br>
+
+## Motivation
+
+**Cyber Threat Intelligence** (CTI) provides a framework for threat analysts to document the operations of a threat actor group, and record the findings of their investigations of specific cyber attack incidents.
+
+With the increasing number and sophistication of attacks occuring across organization's workspace, CTI allows organisations to:
+- Develop a more robust and proactive security posture 
+- Better detect threat vulnerabilities in their infrastructre 
+- Adopt security solutions and policies that allow them to better protect their environment. 
+
+For example **Indicators of Compromise (IoC)** represent network artifacts of a cyber intrusion and are widely used in intrusion detection systems and antivirus softwares to detect future attacks.
+
+**Threat Intel Data** is another form of CTI which comprises of unstructured text data, describing the tools, techniques and procedures (TTPs) used by threat actor groups in a cyber operation. Historically TI data is made available to the security community in the form of *blog posts reports* and *white papers*. 
+
+With the increasing numebr of cyber attacks, it is not scalable to manually process this growing corpus of TI data to understand the motivations capabilities and TTPs associated with an actor group. Additionally TI data does not facilitate easy extraction of IoCs which, if documented in the report, can result in the loss of known indicators in the threat intelligence corpus. 
```

Comment:
You use 'With the increasing numebr of cyber attacks' in the paragraph before - you could remove it here to improve readability

Reply:
Resolved